proto-lens-tests/tests/any.proto

```diff
@@ -10,3 +10,8 @@ message Bar {
   int32 a = 1;
   string b = 2;
 }
+
```

Comment:
Move this to a new unit test `service_test` with corresponding file `service.proto`.